File: app/src/main/java/com/jchshi/readalong/AppConfig.java
package com.jchshi.readalong;

import java.util.Arrays;
import java.util.List;

public final class AppConfig {
    public static final List<WebEntry> SITES = Arrays.asList(
        new WebEntry("Read Along", "https://readalong.google.com/")
    );

    public static final int DEFAULT_SITE_INDEX = 0;

    private AppConfig() {
    }
}

File: app/src/main/java/com/jchshi/readalong/AppConfig.kt
package com.jchshi.readalong

data class WebEntry(
    val title: String,
    val url: String,
)

object AppConfig {
    val sites = listOf(
        WebEntry("Read Along", "https://readalong.google.com/"),
    )

    const val defaultSiteIndex = 0
}
